libraries/timeline-view-compose/src/main/java/com/trendyol/uicomponents/timelineviewcompose/ScaleAnimation.kt

```diff
@@ -0,0 +1,43 @@
+package com.trendyol.uicomponents.timelineviewcompose
+
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.graphics.graphicsLayer
+import com.trendyol.uicomponents.timelineviewcompose.model.PointAnimation
+import kotlinx.coroutines.delay
+
+@Stable
+internal fun Modifier.scaleAnimation(pointAnimation: PointAnimation?) = composed {
+
+    if (pointAnimation == null) return@composed this
+
+    var shouldStartAnimation by remember { mutableStateOf(false) }
+    val infiniteTransition = rememberInfiniteTransition()
+    val pointScale = if (shouldStartAnimation) {
+        infiniteTransition.animateFloat(
+            initialValue = pointAnimation.initialValue,
+            targetValue = pointAnimation.targetValue,
+            animationSpec = pointAnimation.animationSpec,
+        )
+    } else null
+
+    LaunchedEffect(Unit) {
```

Comment:
It's just note for us. Could we write wrapper remember to do like operation because we have many cases like this after different delay we have to trigger any thing. We can talk.